src/open_inwoner/plans/tests/test_views.py

```diff
@@ -329,6 +332,55 @@ def test_plan_file_add_contact_can_access(self):
 
         self.assertEqual(self.plan.documents.count(), 1)
 
+    def test_plan_note_edit_saves_note_correctly(self):
+        response = self.app.get(self.note_edit_url, user=self.user)
+        self.assertEqual(response.status_code, 200)
+
+        form = response.forms["note-edit"]
+        form["note"] = "<p>Form-based test note.</p>"
+
+        response = form.submit(user=self.user)
+        self.assertEqual(response.status_code, 302)
+
+        self.plan.refresh_from_db()
+        self.assertIn("Form-based test note", self.plan.note)
+
+    def test_plan_note_edit_appends_content(self):
+        # Step 1: First edit
+        response = self.app.get(self.note_edit_url, user=self.user)
+        self.assertEqual(response.status_code, 200)
+        form = response.forms["note-edit"]
+        form["note"] = "<p>First note content.</p>"
+        response = form.submit(user=self.user)
+        self.assertEqual(response.status_code, 302)
+
+        self.plan.refresh_from_db()
+        self.assertIn("First note content", self.plan.note)
+
+        # Step 2: Second edit (append new content)
+        response = self.app.get(self.note_edit_url, user=self.user)
+        form = response.forms["note-edit"]
+        # Simulate appending to existing note
+        form["note"] = self.plan.note + "<p>Second note content.</p>"
+        response = form.submit(user=self.user)
+        self.assertEqual(response.status_code, 302)
+
+        self.plan.refresh_from_db()
+        self.assertIn("First note content", self.plan.note)
+        self.assertIn("Second note content", self.plan.note)
+
+    def test_plan_note_edit_contact_can_edit(self):
```

Comment:
The test should include a note by a citizen, then a note by the contact, and check that both contents are present.